dokka-subprojects/plugin-base-frontend/src/main/ui-kit/dropdown/index.ts

```diff
@@ -79,6 +99,10 @@ function addKeyboardNavigation(dropdown: HTMLElement): void {
       (dropdown.querySelector(DROPDOWN_TOGGLE) as HTMLElement)?.focus();
     }
   });
+  const dropdownOptions = dropdown.querySelectorAll('.dropdown--option');
+  dropdownOptions.forEach((option: Element) => {
+    option.addEventListener('keydown', preventScrollBySpaceKey);
+  });
```

Comment:
I'd add the event listener via event delegation to reduce listeners (but, if the number of options is low we can leave it as is).

Reply:
the amount of the option is not too high but I changed it anyway